rear_lidar_node:
  ros__parameters:
    lidar_frame: rear_lidar_link
    start_horizontal_angle: 0.0
    end_horizontal_angle: 6.283185307179586
    max_distance: 100.0
    min_distance: 1.0
    horizontal_resolution: 0.017453292519943295
    vertical_angles: [
      -0.2617993877991494,
      -0.22689280275926285,
      -0.19198621771937624,
      -0.15707963267948966,
      -0.12217304763960307,
      -0.08726646259971647,
      -0.05235987755982989,
      -0.017453292519943295,
      0.017453292519943295,
      0.05235987755982989,
      0.08726646259971647,
      0.12217304763960307,
      0.15707963267948966,
      0.19198621771937624,
      0.22689280275926285,
      0.2617993877991494
      ]
    objects_filename: objects.json
